opensrp-chw/src/main/java/org/smartregister/chw/dao/ReportDao.java

```diff
@@ -51,6 +51,14 @@
                 "left join ec_family f on c.relational_id = f.base_entity_id " +
                 "inner join ec_family_member_location l on l.base_entity_id = c.base_entity_id " +
                 "where ( l.location_id IN " + _communityIds + " or '" + communityIds.get(0) + "' = '') " +
+                " AND CASE WHEN c.gender = 'Male' \n" +
```

Comment:
Execute this query with and without the clauses you added and confirm that they are functional

Reply:
Break you query into legible clauses, its impossible to determine what the clauses are doing

Reply:
The case cause introduced does not  have any effect on the data